### ISSUE TITLE
Repeated measures plot visibility

### ISSUE THREAD

**Issue by TomiSzucs** [cogstat/cogstat#229]
Currently displaying a repeated measures plot, especially with the seaborn_dark theme, the lines connecting related data points are barely visible.

**Comment by krajcsi** [comment]
Yes, more generally, we should find an overall strategy how to handle custom colors to be in line with any themes.

Although, I guess the chosen gray is not hard to see with the default theme (seaborn muted). It is intentionally not a strong color - we may reconsider this too.
